spec/datadog/profiling/collectors/info_spec.rb

```diff
@@ -105,6 +105,28 @@
         end
       end
     end
+
+    describe "gc tuning reporting" do
+      context "when no gc tuning env vars are set" do
+        it "reports an empty hash" do
+          expect(info.fetch(:runtime).fetch(:gc_tuning)).to eq({})
```

Comment:
Not much, but shorter


```suggestion
          expect(info.dig(:runtime, :gc_tuning)).to eq({})
```

Reply:
Actually I did do it like this on purpose -- I wanted a `KeyError: key not found: :gc_tuning (KeyError)` (or similar) if it failed, rather than a `nil does not eq {}`. Small but... why not ;)

Reply:
Fair enough 👍🏼